serverless.yml

```diff
@@ -1,5 +1,10 @@
 service: aws-node-sample-transcribe-s3
 
+package:
+  exclude:
```

Comment:
The `exclude` directive is [discouraged](https://www.serverless.com/framework/docs/deprecations/#NEW_PACKAGE_PATTERNS) (kind of deprecated but then re-added for legacy purposes).  The Serverless Framework recommends adopting the new `patterns` directive: https://www.serverless.com/framework/docs/providers/aws/guide/packaging

Can we just ignore everything and only include `src`? Something like this:

```yaml
  patterns:
    - '!**' # not 100% sure about this syntax
    - 'src/**'
```

This would also clarify another implicit change. By ignoring `node_modules`, we rely on the aws-sdk shipped in the Lambda runtime. It's ok but helps to understand if future Lambda runtimes break the sample once they ship a newer SDK.

Reply:
thanks @joe4dev, I have made the required changes.